packages/protobuf/src/message.ts

```diff
@@ -142,8 +142,10 @@ export class Message<T extends Message<T> = AnyMessage> {
    * message type.  As a result, attempting to serialize a message with this
    * type will throw an Error.
    *
-   * Note that this method is protected because users should not need to invoke
-   * it directly -- they can use JSON.stringify.
+   * This method is protected because users should not need to invoke it
+   * directly -- they should use JSON.stringify or toJsonString for
+   * stringified JSON.  Alternatively, if actual JSON is desired, users should
+   * use toJson.
```

Comment:
All JSDoc is documentation for our users. As a reader, it gives _you_ documentation on how to use it. WDYT about addressing users with "you" consistently?

Reply:
Works for me.  Updated.